bin/hermes

```diff
@@ -32,7 +33,15 @@ def request_get(path):
     Returns:
         the http response
     """
-    response = requests.get(settings.hermes_server + path)
+    retries = 0
+    clear_response = False
+    while not clear_response and retries < 5:
```

Comment:
Are you sure you want 5 retries hard-coded? I see you've already got client settings, why not make it configurable?